src/libraries/System.Private.CoreLib/src/System/Numerics/VectorMath.cs

```diff
@@ -27,5 +29,26 @@ public static bool NotEqual(Vector128<float> vector1, Vector128<float> vector2)
             Debug.Assert(Sse.IsSupported);
             return Sse.MoveMask(Sse.CompareNotEqual(vector1, vector2)) != 0;
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector128<T> ConditionalSelectBitwise<T>(Vector128<T> selector, Vector128<T> ifTrue, Vector128<T> ifFalse)
+            where T : struct
+        {
+            Debug.Assert(Sse.IsSupported || AdvSimd.IsSupported);
+
+            if (Sse.IsSupported)
+            {
+                return Sse.Or(
+                            Sse.And(ifTrue.AsSingle(), selector.AsSingle()),
+                            Sse.AndNot(selector.AsSingle(), ifFalse.AsSingle())
+                        ).As<float, T>();
```

Comment:
This should probably have an SSE2 variant that emits `pand`/`pandn`/`por` for integral `T` to avoid potential bypass delays.

And be sure you check the codegen with the helper to make sure it doesn't regress.  I have a similar helper and had to pass the args by ref (`in` is fine) to get the JIT to not make extra copies.

Reply:
Checked and for the copysign method there is no regression. Single mem access to load the constant and that's it. Will add integral types now

Reply:
I'd recommend just exposing explicit `float` and `double` (rather than generic) versions today.
We don't have active need for the `integral` types and they can be added when needed later instead.
We've also avoided generics for most of the `Vector128` types so far, because the list of supported types is small/well-defined and not all operations are available for all types.

Reply:
Done :+1: